website/docs/docs/build/python-models.md

```diff
@@ -221,6 +241,29 @@ def model(dbt, session):
 
 </File>
 
+<VersionBlock firstVersion="1.8">
+    
+
+#### Dynamic configurations
+
+In addition to the existing methods of configuring Python models, you also have dynamic access to configuration values set with `dbt.config()` within Python models using f-strings. This expands the possibilities for custom logic and configuration management.
+
+<File name='models/my_python_model.py'>
+
+```python
+def model(dbt, session):
+    dbt.config(materialized="table")
+    
+    # Dynamic configuration access within Python f-strings, which allows for real-time retrieval and use of configuration values.
```

Comment:
sugg: making this sentence span 2 lines. that way the horizontal scrollbar doesn't show up. 

imho, h-scrollbars are totally fine if it's code and we can't do anything about it, however, for comments i'd lean towards fewer mouse clicks